app/src/test/report/components/Reports/ReportListComponentTests.tsx

```diff
@@ -1,30 +1,21 @@
 import * as React from "react";
 import { expect } from "chai";
 import { shallow } from "enzyme";
-import {alt} from "../../../../main/shared/alt";
 import { mockReport } from "../../../mocks/mockModels";
 import {ReportListComponent} from "../../../../main/report/components/Reports/ReportList";
 import {ReportListItem} from "../../../../main/report/components/Reports/ReportListItem";
 
 describe("ReportListComponent", () => {
     it("can get props from stores", () => {
-        const reports = [ "report1", "report2" ];
-        alt.bootstrap(JSON.stringify({
-            ReportStore: {
-                ready: true,
-                reports: reports
-            }
-        }));
-
-        expect(ReportListComponent.getPropsFromStores()).to.eql({
-            ready: true,
-            reports: reports
-        });
+        const reports = [ mockReport(), mockReport() ];
```

Comment:
The name of this test should probably be changed to just `renders reports`? And then we need to test the `map...` functions that actually map the props from the global state.